.travis.yml

```diff
@@ -32,18 +27,14 @@ jobs:
         - npm install --no-audit
         - ln -s /dev/stdout ./lerna-debug.log
         - npm run install-locks
-        - echo 'BOOTSTRAP FINISHED'
       script: ./travis-ci/audit-packages.sh
 
     - name: "eslint"
       stage: unit test and stack deployment
       install:
-        - ./travis-ci/fetch-cache.sh
         - npm install
         - ln -s /dev/stdout ./lerna-debug.log
         - npm run bootstrap-no-build
-        - env SKIP_BUILD=true ./bin/prepare
```

Comment:
How do we not need this anymore? We always want our integration build to use the latest changes to the packages that we've committed, so I would think this is necessary

Reply:
Adding `example` to `lerna.json` gets us that.